keyboards/mechlovin/hannah65/rules.mk

```diff
@@ -19,4 +19,4 @@ RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
 BLUETOOTH_ENABLE = no       # Enable Bluetooth
 AUDIO_ENABLE = no           # Audio output
 
-DEFAULT_FOLDER = mechlovin/hannah65/rev1
\ No newline at end of file
+DEFAULT_FOLDER = mechlovin/hannah65/rev1/haus
```

Comment:
I'm really fond of this, as it changes the outcome of `qmk compile -kb mechlovin/hannah65 -km default`.